### PR TITLE
add log for attach probe failed

### PR DESCRIPTION
**Pull request by jichenjc**
Related to #847

### CODE REVIEW

**Review by aspanner — APPROVED** [review]
lgtm

**Review by aspanner — APPROVED** [review]
can you add the variable output as well?

**Comment by vimalk78** [comment]
just adding some context to the function `finish_task_switch.isra.0` 

`isra.NNN` is the suffix added to the function name when gcc option `-fipa-sra` compiler optimization being carried out.

https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html
```
-fipa-sra

    Perform interprocedural scalar replacement of aggregates, removal of unused parameters and replacement of parameters passed by reference by parameters passed by value.

    Enabled at levels -O2, -O3 and -Os.
```
So kepler looks for a function with `isra.0` suffix to attach kprobe. But why only `isra.0` ? should kepler also look for other suffixes, `isra.1`, `isra.2` ? 
Cc: @rootfs @marceloamaral

File: pkg/bpfassets/attacher/bcc_attacher.go [hunk 1]
```diff
@@ -98,6 +98,7 @@ func loadBccModule(objProg []byte, options []string) (m *bpf.Module, err error)
 	}
 	err = m.AttachKprobe("finish_task_switch", ftswitch, -1)
 	if err != nil {
+		klog.Infof("finish_task_switch failed, try finish_task_switch.isra.0 instead")
```
Comment (vimalk78) [inline pkg/bpfassets/attacher/bcc_attacher.go:101]:
```suggestion
		klog.Infof("attaching kprobe to finish_task_switch failed, trying finish_task_switch.isra.0 instead")
```

**Review by rootfs — APPROVED** [review]

**Pull request merged**